scripts/scenarios/cloud6/qa-scenario-2b-sbd-kvm-vmware.yaml

```diff
@@ -131,6 +131,8 @@ proposals:
         host: vcs.qa.suse.de
         user: "@@vcenter_user@@"
         password: "@@vcenter_password@@"
+        cluster_name: []
```

Comment:
@mjura I've no idea what to put here for the job that is running on qa hardware. Do you know?